ui/src/app/state/user/user.actions.ts

```diff
@@ -0,0 +1,4 @@
+import { createAction, props } from "@ngrx/store";
+import { User } from "../../services";
+
+export const setUser = createAction('[User] Set', props<{ user: User | undefined }>());
```

Comment:
any opinion on  using action groups?
might be overkill for this task, but for larger apps a nice thing, hence might be nice to show it exists.
https://ngrx.io/guide/store/action-groups